#include "team_server_player.h"
#include "team_server_player.h"
#include "team_server_player.h"
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE

namespace serverplayer{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE

///<<<rpc begin
void PlayerPlayerTeamServiceImpl::EnterScene(common::EntityPtr& entity,
    const serverplayer::TeamTestRequest* request,
    serverplayer::TeamTestResponse* response)
{
///<<< BEGIN WRITING YOUR CODE EnterScene
///<<< END WRITING YOUR CODE EnterScene
}

///<<<rpc end
}// namespace serverplayer
